.github/workflows/submitty_ci.yml

```diff
@@ -105,7 +106,7 @@ jobs:
                 working-directory: site
         steps:
             - uses: actions/checkout@v3
-            - uses: shivammathur/setup-php@2.25.2
+            - uses: shivammathur/setup-php@2.25.1
```

Comment:
It looks like this dependency has been updated since you created this PR.  Please bump the version to 2.25.2 like we have on main.